test/tests/functional/pbs_reservations.py

```diff
@@ -90,20 +90,34 @@ def submit_asap_reservation(self, user, jid, extra_attrs=None):
 
         return self.server.submit(r)
 
-    @skipOnCpuSet
```

Comment:
Why are you removing this? This would cause the tests that call this function to fail on cpuset.

Reply:
It's a helper function, not a test.  The tests do have the decorator,

Reply:
I'm on master and I don't see the tests having the decorator. That being said, I'm not sure if the decorator works in a helper function. Should you put the decorator on the tests?

Reply:
Yeah, I must have been looking at different tests.  I've added the skip decorator now.